docs/modules/ROOT/pages/erc721.adoc

```diff
@@ -1,772 +1,300 @@
-
-Parameters:
-
-[,cairo]
-----
-owner: felt
-operator: felt
-----
-
-Returns:
-
-[,cairo]
-----
-approved: felt
-----
-
-=== Events
-
-==== `Approval (Event)`
-
-Emitted when `owner` enables `approved` to manage the `tokenId` token.
-
-Parameters:
-
-[,cairo]
-----
-owner: felt
-approved: felt
-tokenId: Uint256
-----
-
-==== `ApprovalForAll (Event)`
-
-Emitted when `owner` enables or disables (`approved`) `operator` to manage all of its assets.
-
-Parameters:
-
-[,cairo]
-----
-owner: felt
-operator: felt
-approved: felt
-----
-
-==== `Transfer (Event)`
-
-Emitted when `tokenId` token is transferred from `from_` to `to`.
-
-Parameters:
-
-[,cairo]
-----
-from_: felt
-to: felt
-tokenId: Uint256
-----
-
-'''
-
-=== IERC721Metadata API
-
-[,cairo]
-----
-func name() -> (name: felt) {
-}
-
-func symbol() -> (symbol: felt) {
-}
-
-func tokenURI(tokenId: Uint256) -> (tokenURI: felt) {
-}
-----
-
-==== `name`
-
-Returns the token collection name.
-
-Parameters: None.
-
-Returns:
-
-[,cairo]
-----
-name: felt
-----
-
-==== `symbol`
-
-Returns the token collection symbol.
-
-Parameters: None.
-
-Returns:
-
-[,cairo]
-----
-symbol: felt
-----
-
-==== `tokenURI`
-
-Returns the Uniform Resource Identifier (URI) for `tokenID` token.
-If the URI is not set for the `tokenId`, the return value will be `0`.
-
-Parameters:
-
-[,cairo]
-----
-tokenId: Uint256
-----
-
-Returns:
-
-[,cairo]
-----
-tokenURI: felt
-----
-
-'''
-
-=== IERC721Enumerable API
-
-[,cairo]
-----
-func totalSupply() -> (totalSupply: Uint256) {
-}
-
-func tokenByIndex(index: Uint256) -> (tokenId: Uint256) {
-}
-
-func tokenOfOwnerByIndex(owner: felt, index: Uint256) -> (tokenId: Uint256) {
-}
-----
-
-==== `totalSupply`
-
-Returns the total amount of tokens stored by the contract.
-
-Parameters: None
-
-Returns:
-
-[,cairo]
-----
-totalSupply: Uint256
-----
-
-==== `tokenByIndex`
-
-Returns a token ID owned by `owner` at a given `index` of its token list.
-Use along with <<balanceof,balanceOf>> to enumerate all of ``owner``'s tokens.
-
-Parameters:
-
-[,cairo]
-----
-index: Uint256
-----
-
-Returns:
-
-[,cairo]
-----
-tokenId: Uint256
-----
-
-==== `tokenOfOwnerByIndex`
-
-Returns a token ID at a given `index` of all the tokens stored by the contract.
-Use along with <<totalsupply,totalSupply>> to enumerate all tokens.
+Token URIs in Cairo are stored as single field elements.
+Each field element equates to 252-bits (or 31.5 bytes) which means that a token's URI can be no longer than 31 characters.
 
-Parameters:
+NOTE: Native string support in Cairo is currently in progress and tracked {string-roadmap}.
+Once Cairo offers full string support, this will be revisited.
 
-[,cairo]
-----
-owner: felt
-index: Uint256
-----
+== Presets
 
-Returns:
+ERC721 presets have been created to allow for quick deployments as-is whic are a great option for testing and prototyping.
 
-[,cairo]
-----
-tokenId: Uint256
-----
+== Extensions
```

Comment:
I think we should mention this interface on top as mentioned in a comment, and we can remove this section.

Reply:
The IERC721Metadata interface I mean.